h/script.py

```diff
@@ -197,19 +198,15 @@ def token(args):
     X-Annotator-Auth-Token header.
 
     """
-    from h.auth import get_client, generate_signed_token
-
+    import h.api.auth
     request = bootstrap(args)
-    registry = request.registry
-
-    request.client = get_client(request, registry.settings['h.client_id'])
-    request.user = args.sub
-    request.expires_in = args.ttl
-    request.extra_credentials = {}
-
-    token = generate_signed_token(request)
+    mock_request = mock.Mock(
```

Comment:
Using `mock` here is a bit weird. If you have to use a fake request object, perhaps use a `namedtuple` instead.